python/pacemaker/_cts/audits.py

```diff
@@ -122,10 +122,11 @@ def _test_logging(self):
 
         watch_pref = self._cm.env["LogWatcher"]
         if watch_pref == LogKind.ANY:
-            kinds = [LogKind.FILE]
```

Comment:
Meaning a program that gathers logs in real time from remote nodes and writes them locally? Didn't think about that.

FWIW, this only works if an aggregator happens to write to the local `/var/log/messages`. As soon as you specify a log file name, `cts-lab` will look only on the remote nodes.

That could be controlled better with a new CLI option. I'm just pointing out that the current state of aggregator support is:
* minimal
* undocumented

Reply:
Yep, that's the usual setup -- rsyslog (and others I'm sure) allows aggregating logs from multiple hosts, typically into the aggregator's local log. It would be better if the location were configurable, but we can leave it be

Reply:
(Note for posterity: "Drop this commit' was from a previous iteration of this commit that did something different. The comment doesn't show a "Outdated" because of where it was placed.)